### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Marianna-Weychan**
ok group!
I made a preliminary App.js template (it needs to be supplemented with imports),
in addition, I created the file queryParamsName.js, route.js and svg icons, please check if everything is fine and I'm still working!

### CODE REVIEW

File: src/core/App/App.js [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+        <Route path={toPerson()}>
+          <ProfilePersonPage />
+        </Route>
+        <Route path={toPeople()}>
+          <PeopleList />
+        </Route>
+        <Route>
+          <Redirect to={toMovies()} />
+        </Route>
+      </Switch>
```
Comment (AdrianJar) [inline src/core/App/App.js:26]: I think we should make new file for this part  and import it to APP.

**Review by AdrianJar — CHANGES_REQUESTED** [review]
Good job, i think we should consider what do to with my comment

File: src/core/App/App.js (hunk 1 above)
Comment (ghost) [reply src/core/App/App.js:26]: You're right, in the afternoon at the meeting we will determine the appearance of App.js and make changes and remove the conflict in merge

**Review by AdrianJar — APPROVED** [review]
Good job! Everything done well

**Review by KatarzynaZaloba — APPROVED** [review]
I see, nice job, Marianna!

**Pull request merged**